samcli/hook_packages/terraform/hooks/prepare/resource_linking.py

```diff
@@ -2,3 +2,41 @@
 Use Terraform plan to link resources together
 e.g. linking layers to functions
 """
+
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional
+
+
+@dataclass
+class Value:
+    constant_value: Any
+    references: List[str]
+
+
+@dataclass
+class Module:
+    # full path to the module, including parent modules
+    full_address: str
+    parent_module: Optional["Module"]
```

Comment:
have to either put it in quotes like this or do `from __future__ import annotations`, in order to self reference like this.
Preference on any way?

Reply:
I would say let's aim for consistency through the CLI. We've typically been doing it this way so far so let's just keep going with it unless anyone else feels strongly about it.